disperser/batcher/encoding_streamer.go

```diff
@@ -137,7 +137,11 @@ func (e *EncodingStreamer) Start(ctx context.Context) error {
 						// ignore canceled errors because canceled encoding requests are normal
 						continue
 					}
-					e.logger.Error("error processing encoded blobs", "err", err)
+					if strings.Contains(err.Error(), "too many requests") {
+						e.logger.Warn("error processing encoded blobs", "err", err)
+					} else {
+						e.logger.Error("error processing encoded blobs", "err", err)
```

Comment:
I feel like it could be confusing to use the same message for a Warning (which is safe to ignore) vs Error which are not safe to ignore. Can we change the error message in a way that is more provocative call to action.

Reply:
Updated the message for warning